script/lint.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
 set -exou pipefail
-git grep -l "bin\/bash" | xargs shellcheck -x -f gcc
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+
+git grep -l "bin\/bash" | xargs shellcheck -x -f gcc --source-path "$GIT_ROOT"
```

Comment:
The `--source-path` change makes it so this script isn't dependant on the cwd for the purposes of finding the sourced files.

This DOES however require a relatively new version of shellcheck (I had to upgrade locally to get this option). So if that doesn't work or isn't required I can remove this too

Reply:
Removed this, cause I realized other scripts needs to be run repo root anyways for other reasons, so no a super strong need for this

But if you actually do want it just let me know lol